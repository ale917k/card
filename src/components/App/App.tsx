import { FC } from "react";
import { Routes, Route } from "react-router-dom";

import Home from "pages/Home";

/**
 * Main entry point containing navigation routes.
 */
const App: FC = () => (
  <Routes>
    <Route path="*" element={<Home />} />
  </Routes>
);

export default App;
