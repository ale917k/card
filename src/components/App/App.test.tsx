import { shallow } from "enzyme";

import App from "./App";

describe("App", () => {
  it("Renders main element", () => {
    const link = shallow(<App />);
    expect(link).toMatchSnapshot();
  });
});
