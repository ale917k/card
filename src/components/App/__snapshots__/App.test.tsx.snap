// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Renders main element 1`] = `
<div>
  App
</div>
`;
